LinkTo frozen class

Runing linkto in certain conditions, I got this error:

```
Caused by: java.lang.RuntimeException: CustomDogController$linkTo: frozen class (cannot edit)
    at javassist.ClassPool.checkNotFrozen(ClassPool.java:616) [javassist-3.18.1-GA.jar:3.18.0-GA]
    at javassist.ClassPool.makeInterface(ClassPool.java:854) [javassist-3.18.1-GA.jar:3.18.0-GA]
    at javassist.ClassPool.makeInterface(ClassPool.java:839) [javassist-3.18.1-GA.jar:3.18.0-GA]
    at br.com.caelum.vraptor.view.LinkToHandler.createLinkToInterface(LinkToHandler.java:151) [vraptor-4.0.0-beta-4-SNAPSHOT.jar:]
    at br.com.caelum.vraptor.view.LinkToHandler.get(LinkToHandler.java:123) [vraptor-4.0.0-beta-4-SNAPSHOT.jar:]
    at br.com.caelum.vraptor.view.LinkToHandler$Proxy$_$$_WeldClientProxy.get(Unknown Source) [vraptor-4.0.0-beta-4-SNAPSHOT.jar:]
```

--- a/vraptor-core/src/main/java/br/com/caelum/vraptor/view/LinkToHandler.java
+++ b/vraptor-core/src/main/java/br/com/caelum/vraptor/view/LinkToHandler.java
@@ -299,7 +299,7 @@ public class LinkToHandler extends ForwardingMap<Class<?>, Object> {
 			CtClass ctClass = pool.getOrNull(clazz.getName());
 			if (ctClass != null) {
 				ctClass.detach();
-				logger.debug("class {} is detached");
+				logger.debug("class {} is detached", clazz.getName());
 			}
 		}
 	}

